.github/workflows/flair.yml

```diff
@@ -36,9 +36,6 @@ jobs:
       - name: Run unit tests
         run: |
           bash scripts/gh-action-unit-test.sh ${GITHUB_WORKFLOW}
-      - name: Run integration tests
```

Comment:
Why disable the flair and estuary integration tests?

Reply:
Looks like neither of those actually have any integration tests to run